packages/web-components/fast-foundation/src/accordion/accordion.ts

```diff
@@ -111,18 +110,28 @@ export class FASTAccordion extends FASTElement {
             item.addEventListener("keydown", this.handleItemKeyDown);
             item.addEventListener("focus", this.handleItemFocus);
         });
+
         if (this.isSingleExpandMode()) {
-            const expandedItem: FASTAccordionItem | null =
-                this.findExpandedItem() ??
-                ((this._accordionItems[0] as unknown) as FASTAccordionItem);
-            expandedItem.setAttribute("aria-disabled", "true");
+            const expandedItem = this.findExpandedItem() as FASTAccordionItem;
+            this.setSingleExpandMode(expandedItem);
         }
     };
 
-    private resetItems(): void {
-        // reset ALL items if we swap to single expand mode (should we reset an expanded item which is disabled?)
-        this.accordionItems.forEach((item: FASTAccordionItem, index: number) => {
-            item.expanded = false;
+    private setSingleExpandMode(expandedItem: Element): void {
+        const currentItems = Array.from(this._accordionItems);
+        this.activeItemIndex = currentItems.indexOf(expandedItem);
+
+        currentItems.forEach((item: FASTAccordionItem, index: number) => {
+            if (this.activeItemIndex === index) {
+                item.expanded = true;
+                item.expandbutton.setAttribute("aria-disabled", "true");
```

Comment:
Can this be handled with an observable on the `disabled` property in the `accordion-item`?

Reply:
I don't think so for this scenario as it's somewhat unique. This is specifically different than a _disabled_ item holistically and disabling the _button_ of the active item in single-expand mode.